pkg/wire/util/uint160.go

```diff
@@ -65,7 +65,7 @@ func (u Uint160) BytesReverse() []byte {
 
 // String implements the stringer interface.
 func (u Uint160) String() string {
-	return hex.EncodeToString(u.Bytes())
+	return hex.EncodeToString(u.BytesReverse())
```

Comment:
This was left as u.Bytes(), so that we had to be explicit in the endian that we wanted. For example; If we wanted little endian, then we should first reverse, then call String(). Having it this way, allows us to use util.Uint160 everywhere without worrying about the endian switching.

A smaller negligible side-effect: This also allows us to not worry about the case where we call String() which reverses it, then we instantiate another uint160 with that string. Although this could be solved by having the New function take it in LE, but then we would no longer need the BytesReverse. A ReverseString method would be better imo.

Reply:
The main goal is internal endian consistency, we can always have interoperability with the external APIs

Reply:
:+1:  
I will revert the changes and add `ReverseString` method.

Reply:
If I stick to your approach, shouldn't we change this as well? (It was already there in the dev branch: [code](https://github.com/CityOfZion/neo-go/blob/dev/pkg/wire/util/uint256.go#L64))
```go
// String implements the stringer interface.
func (u Uint256) String() string {
	return hex.EncodeToString(slice.Reverse(u.Bytes()))
}
```

Reply:
Yep didn't catch that